### PR TITLE
Draft of September Heartbeat

### PR DESCRIPTION
**Pull request by elleobrien**
ready for review! did I leave out any news? Let me know and I'm happy to add it.

### CODE REVIEW

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+Last week, we held a meetup for contributors to DVC! Core maintainer
+[Ruslan Kupriev](https://github.com/efiop) hosted a get-together for folks who
+contribute new features, bug fixes, and more to the community. If you missed it,
+you can watch it on YouTube.
+
+https://youtu.be/jUYSTERXxWg
+
+### New videos
+
+We've released several new videos to our growing
```
Comment (dmpetrov) [inline content/blog/2020-09-09-september-20-dvc-heartbeat.md:61]: You can emphasize even more on the success - like "we recently started blogging and have already reached 1000 subscribers" or more personal "I remember our first video tutorial - it was a month ago but I remember it like yesterday". 

You can also mention how great this youtube community is - comment, same users show up over and over again in the comments.

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+DVC ambassador [Marcel Ribeiro-Dantas](https://twitter.com/mribeirodantas) has
+published two of three tutorial blogs in a series on CML! Marcel's use case is
+especially cool because he's using R, plus some causal modeling related to his
+work in bioinformatics, with GitHub Actions.
+
+In Part I, Marcel introduces his project and how he uses DVC, CML and GitHub
+Actions together (with his custom R library).
+
+<external-link
```
Comment (dmpetrov) [inline content/blog/2020-09-09-september-20-dvc-heartbeat.md:96]: An issue with rendering the image.
Screen Shot 2020-09-10 at 11 54 44 AM

**Review by dmpetrov — APPROVED** [review]
Amazing ✨ 
Just one minor comment and an image issue.

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md (hunk 2 above)
Comment (elleobrien) [reply content/blog/2020-09-09-september-20-dvc-heartbeat.md:99]: should be resolved now

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md (hunk 1 above)
Comment (elleobrien) [reply content/blog/2020-09-09-september-20-dvc-heartbeat.md:61]: yep, i expanded it!

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+title: September '20 Heartbeat
+date: 2020-09-09
+description: |
+  This month, catch us on the Software Engineering Daily Podcast, 
+  check out our favorite DVC and CML tutorials and projects, and 
+  celebrate 1000 YouTube subscribers!
+descriptionLong: |
+  This month, catch us on the Software Engineering Daily Podcast, 
+  check out our favorite DVC and CML tutorials and projects, and 
+  celebrate 1000 YouTube subscribers!
```
Comment (shcheklein) [inline content/blog/2020-09-09-september-20-dvc-heartbeat.md:11]: I guess we can make YouTube a link to get even more subscribers - DescriptionLong should support that. Also we can make DVC and CML links?

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - R
+  - Meetup
+  - Videos
+---
+
+Welcome to our September roundup of cool news, new releases, and recommended
```
Comment (shcheklein) [inline content/blog/2020-09-09-september-20-dvc-heartbeat.md:24]: It feels we can skip this paragraph? It feels repetitive ... may be even next two titles? (not sure)

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+## News
+
+### Dmitry on Software Engineering Daily
+
+Our CEO Dmitry Petrov was interviewed on the much-beloved Software Engineering
+Daily podcast! Host [Jeff Meyerson](https://twitter.com/the_prion) kicked off
+the discussion:
+
+> Code is version controlled through git, the version control system originally
```
Comment (shcheklein) [inline content/blog/2020-09-09-september-20-dvc-heartbeat.md:35]: 💅 git -> Git

**Review by shcheklein — APPROVED** [review]
Looks awesome! A few comments + Real_Python.png can be made smaller I hope.

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md (hunk 4 above)
Comment (elleobrien) [reply content/blog/2020-09-09-september-20-dvc-heartbeat.md:24]: yeah, let's skip it. it felt silly writing it, i think that's a sign it's repetitive.

File: content/blog/2020-09-09-september-20-dvc-heartbeat.md (hunk 5 above)
Comment (elleobrien) [reply content/blog/2020-09-09-september-20-dvc-heartbeat.md:35]: fixed

**Pull request merged**